package.json

```diff
@@ -18,93 +18,102 @@
       "lib": "src"
     },
     "dependencies": {
-      "@angular/common": "npm:@angular/common@2.0.0-rc.1",
-      "@angular/compiler": "npm:@angular/compiler@2.0.0-rc.1",
-      "@angular/core": "npm:@angular/core@2.0.0-rc.1",
-      "@angular/http": "npm:@angular/http@^2.0.0-rc.1",
-      "@angular/platform-browser": "npm:@angular/platform-browser@2.0.0-rc.1",
+      "@angular/forms": "npm:@angular/forms@^0.1.0",
       "@angular/platform-browser-dynamic": "npm:@angular/platform-browser-dynamic@^2.0.0-rc.0",
-      "@ngrx/core": "npm:@ngrx/core@^1.0.0",
-      "@ngrx/router": "npm:@ngrx/router@^1.0.0-beta.0",
       "@ngrx/router-store": "npm:@ngrx/router-store@^0.0.1",
-      "@ngrx/store": "npm:@ngrx/store@^1.5.0",
       "es6-shim": "github:es-shims/es6-shim@^0.35.0",
       "immutable": "npm:immutable@^3.8.1",
       "lodash": "npm:lodash@^4.11.2",
       "ng2-translate": "npm:ng2-translate@^2.1.0",
+      "plugin-typescript": "github:frankwallis/plugin-typescript@4.0.16",
       "reflect-metadata": "npm:reflect-metadata@^0.1.3",
-      "rxjs": "npm:rxjs@5.0.0-beta.6",
-      "zone.js": "npm:zone.js@0.6.12"
+      "store": "npm:@ngrx/store@^2.0.1"
     },
     "devDependencies": {
-      "assert": "github:jspm/nodelibs-assert@^0.2.0-alpha",
       "json": "github:systemjs/plugin-json@^0.1.2",
       "net": "github:jspm/nodelibs-net@^0.2.0-alpha",
-      "os": "github:jspm/nodelibs-os@^0.2.0-alpha",
-      "path": "github:jspm/nodelibs-path@^0.2.0-alpha",
       "plugin-babel": "npm:systemjs-plugin-babel@^0.0.9",
-      "plugin-typescript": "github:frankwallis/plugin-typescript@^4.0.9",
       "scss": "github:KevCJones/plugin-scss@^0.2.11",
       "systemjs-hot-reloader": "github:capaj/systemjs-hot-reloader@^0.5.7",
-      "tty": "github:jspm/nodelibs-tty@^0.2.0-alpha",
-      "util": "github:jspm/nodelibs-util@^0.2.0-alpha"
+      "tty": "github:jspm/nodelibs-tty@^0.2.0-alpha"
     },
     "peerDependencies": {
+      "@angular/common": "npm:@angular/common@2.0.0-rc.2",
+      "@angular/compiler": "npm:@angular/compiler@2.0.0-rc.2",
+      "@angular/core": "npm:@angular/core@2.0.0-rc.2",
+      "@angular/http": "npm:@angular/http@2.0.0-rc.2",
+      "@angular/platform-browser": "npm:@angular/platform-browser@2.0.0-rc.2",
+      "@ngrx/core": "npm:@ngrx/core@^1.0.0",
+      "@ngrx/router": "npm:@ngrx/router@^1.0.0-beta.0",
+      "@ngrx/store": "npm:@ngrx/store@^2.0.1",
```

Comment:
Why are these _peerDependencies_, whereas @angular/forms isn't? Suppose all of them should be _dependencies_.